### PR TITLE
Changed copy for allowable expenses

### PR DESCRIPTION
**Pull request by Ronolibert**
## Description

Copy changes for allowable expenses page

## Setup
`make server_run`
`make client_run`
`Visit http://milmovelocal:3000/allowable-expenses`

## Code Review Verification Steps
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Pivotal acceptance criteria been met for this change?

## References

* [Pivotal story](https://www.pivotaltracker.com/story/show/166290704) for this change

## Screenshots
image
image

### CODE REVIEW

File: src/scenes/Moves/Ppm/AllowableExpenses.jsx [hunk 1]
```diff
@@ -13,60 +13,91 @@ function AllowableExpenses(props) {
...
+        for authoritative information.
+      </p>
+      <p>
+        <strong>Save your receipts.</strong> It’s better to have receipts you don’t need than to need receipts you don’t
+        have.
       </p>
+      <hr />
+      <FontAwesomeIcon aria-hidden className="icon" icon={faCheck} />Some commonly claimed moving expenses:
+      <ul>
+        <li>Consumable packling materials</li>
```
Comment (LeDeep) [inline src/scenes/Moves/Ppm/AllowableExpenses.jsx]: typo: `packling`

File: src/scenes/Moves/Ppm/AllowableExpenses.jsx [hunk 2]
```diff
@@ -13,60 +13,91 @@ function AllowableExpenses(props) {
...
+        <li>Contracted expenses</li>
+        <li>Oil</li>
+        <li>Rental equipment</li>
+        <li>Tolls</li>
+        <li>Weighing fees</li>
+        <li>Gas, exceeding travel allowance (see below)</li>
+      </ul>
+      <div className="dashed-divider" style={{ marginBottom: '1em' }} />
+      <FontAwesomeIcon aria-hidden className="icon" icon={faBan} />Some common expenses that are not claimable or
```
Comment (LeDeep) [inline src/scenes/Moves/Ppm/AllowableExpenses.jsx]: in the wireframe not is italicized.

Comment (Ronolibert) [reply src/scenes/Moves/Ppm/AllowableExpenses.jsx]: Wow good catch! Fixed this

**Comment by Ronolibert** [comment]
@LeDeep Fixed the typo and italics!

**Pull request merged**